### ISSUE TITLE
Get Started with the Chaos Toolkit (windows)

### ISSUE THREAD

**Issue by gtfisher** [chaostoolkit/chaostoolkit-documentation#53]

**Comment by gtfisher** [comment]
Getting started first experiment gets to "chaos run experiment.json"

when you run this on windows it errors with "path 'None' cannot be found, in activity 'swap-to-expired-cert' "

This is because the swap-to-expired-cert step in the experiment.json  makes reference to "cp" which is not available on all platforms.

**Comment by Lawouach** [comment]
Indeed! It's tricky because the idea of the tutorial is to be simple. But I admit I didn't take Windows as a target platform when I wrote it.
